### PR TITLE
issue 264: local references for internal libraries

### PR DESCRIPTION
**Pull request by a-ajith**

### CODE REVIEW

**Review by irappa-pattar — APPROVED** [review]

**Review by Bharath-KKB — APPROVED** [review]

File: plugin-redfish/go.mod [hunk 1]
```diff
@@ -3,11 +3,21 @@ module github.com/ODIM-Project/ODIM/plugin-redfish
...
 
 require (
-	github.com/ODIM-Project/ODIM/lib-messagebus v0.0.0-20200727133207-df3dfb728bd1
-	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201015111324-83393fdedb82
+	github.com/Joker/hpp v1.0.0 // indirect
+	github.com/ODIM-Project/ODIM/lib-messagebus v0.0.0-20201201072448-9772421f1b55
+	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201201072448-9772421f1b55
 	github.com/gofrs/uuid v3.2.0+incompatible
-	github.com/google/uuid v1.1.2-0.20200519141726-cb32006e483f
+	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
```
Comment (jeevan-kamkar) [inline plugin-redfish/go.mod]: why this package is added

File: plugin-redfish/go.mod [hunk 2]
```diff
@@ -3,11 +3,21 @@ module github.com/ODIM-Project/ODIM/plugin-redfish
...
 	github.com/kataras/iris/v12 v12.1.9-0.20200616210209-a85c83b70ad0
 	github.com/satori/go.uuid v1.2.0
+	github.com/yudai/pp v2.0.1+incompatible // indirect
```
Comment (jeevan-kamkar) [inline plugin-redfish/go.mod]: why this package is added

File: svc-account-session/go.mod [hunk 3]
```diff
@@ -3,10 +3,21 @@ module github.com/ODIM-Project/ODIM/svc-account-session
...
 require (
-	github.com/ODIM-Project/ODIM/lib-persistence-manager v0.0.0-20201021053518-88345d4a9988 // indirect
-	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201015111324-83393fdedb82
+	github.com/Joker/hpp v1.0.0 // indirect
+	github.com/ODIM-Project/ODIM/lib-persistence-manager v0.0.0-20201201072448-9772421f1b55 // indirect
+	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201201072448-9772421f1b55
+	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
 	github.com/satori/go.uuid v1.2.0
 	github.com/stretchr/testify v1.5.1
+	github.com/yudai/pp v2.0.1+incompatible // indirect
```
Comment (jeevan-kamkar) [inline svc-account-session/go.mod]: why this package is added

File: svc-aggregation/go.mod [hunk 4]
```diff
@@ -3,12 +3,23 @@ module github.com/ODIM-Project/ODIM/svc-aggregation
...
-	github.com/ODIM-Project/ODIM/lib-rest-client v0.0.0-20201007090649-06a4c52ce4cf
-	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201016064732-c652b5935f93
+	github.com/Joker/hpp v1.0.0 // indirect
+	github.com/ODIM-Project/ODIM/lib-dmtf v0.0.0-20201201072448-9772421f1b55
+	github.com/ODIM-Project/ODIM/lib-messagebus v0.0.0-20201201072448-9772421f1b55
+	github.com/ODIM-Project/ODIM/lib-persistence-manager v0.0.0-20201201072448-9772421f1b55 // indirect
+	github.com/ODIM-Project/ODIM/lib-rest-client v0.0.0-20201201072448-9772421f1b55
+	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201201072448-9772421f1b55
 	github.com/fsnotify/fsnotify v1.4.9
+	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
```
Comment (jeevan-kamkar) [inline svc-aggregation/go.mod]: why this package is added

File: svc-api/go.mod [hunk 5]
```diff
@@ -3,9 +3,18 @@ module github.com/ODIM-Project/ODIM/svc-api
 go 1.13
 
 require (
-	github.com/ODIM-Project/ODIM/lib-persistence-manager v0.0.0-20201021053518-88345d4a9988 // indirect
-	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201015111324-83393fdedb82
+	github.com/Joker/hpp v1.0.0 // indirect
+	github.com/ODIM-Project/ODIM/lib-persistence-manager v0.0.0-20201201072448-9772421f1b55 // indirect
+	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201201072448-9772421f1b55
+	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
```
Comment (jeevan-kamkar) [inline svc-api/go.mod]: why this package is added

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Please verify why new packages are getting added if it's required please check the licence of them

File: plugin-redfish/go.mod (hunk 1 above)
Comment (a-ajith) [reply plugin-redfish/go.mod]: This package got added after the go mod tidy

File: plugin-redfish/go.mod (hunk 2 above)
Comment (a-ajith) [reply plugin-redfish/go.mod]: This package got added after the go mod tidy

File: svc-aggregation/go.mod (hunk 4 above)
Comment (a-ajith) [reply svc-aggregation/go.mod]: This package got added after the go mod tidy

File: svc-api/go.mod (hunk 5 above)
Comment (a-ajith) [reply svc-api/go.mod]: This package got added after the go mod tidy

**Review by jeevan-kamkar — APPROVED** [review]

**Pull request merged**